src/overlays/actors/ovl_En_Test6/z_en_test6.c

```diff
@@ -821,24 +893,27 @@ void func_80A92188(EnTest6* this, PlayState* play) {
             player->unk_AC0 = 0.0f;
             player->actor.shape.yOffset = 0.0f;
             break;
+
+        default:
+            break;
     }
 
-    if ((this->unk_286 > 0) && (this->unk_286 < 20)) {
-        func_80A90C54(play, this->unk_286 * 0.05f);
-        this->unk_286++;
-        if (this->unk_286 >= 20) {
-            this->unk_27A = 15;
-            this->unk_160 = 0.9333333f;
+    if ((this->screenFillAlpha > 0) && (this->screenFillAlpha < 20)) {
+        EnTest6_EnableWhiteFillScreen(play, this->screenFillAlpha * 0.05f);
+        this->screenFillAlpha++;
+        if (this->screenFillAlpha >= 20) {
+            this->timer = 15;
+            this->doubleSoTEnvLerp = 0.9333333f;
         }
-    } else if ((this->unk_27A < 96) && (this->unk_27A > 50) &&
+    } else if ((this->timer < 96) && (this->timer > 50) &&
                (CHECK_BTN_ALL(input->press.button, BTN_A) || CHECK_BTN_ALL(input->press.button, BTN_B))) {
-        this->unk_286 = 1;
-        this->unk_27A = 39;
+        this->screenFillAlpha = 1;
+        this->timer = 39;
         SEQCMD_STOP_SEQUENCE(SEQ_PLAYER_FANFARE, 20);
     }
```

Comment:
Huh, I didn't remember you could affect this effect by pressing A or B. What does this do?

Reply:
Mashing A/B actually shortens these cutscenes by a little bit. Specifically, the cutscene starts at a timer with a high number, then it counts down to 0 to finish. If the timer is between 96 -> 50, then pressing A/B will skip parts of the cutscene directly to timer 39 and skip the steps in between